### PR TITLE
coc: remove project coc column from table on CPC readme

### PR DESCRIPTION
**Pull request by joesepi**
There were missing CoC links in this table. It was decided to remove the column altogether in favor of folks finding the CoC for each project in the project's org/repo.

related: #1122
closes: #664

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,64 +21,64 @@ The following projects are official OpenJS Foundation projects. If you are inter
...
+| <img alt="logo of project architect" src="https://github.com/openjs-foundation/artwork/raw/HEAD/projects/architect/architect-icon-grayscale.svg" width="30" /> | [architect](https://arc.codes/)         |   |  [Contributing Guide](https://github.com/architect/architect/blob/HEAD/.github/contributing.md)   |
+| <img alt="logo of project ESLint" src="https://github.com/openjs-foundation/artwork/raw/HEAD/projects/eslint/eslint-icon-color.svg" width="30" /> | [ESLint](https://eslint.org/)              |    | [Contributing Guide](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md) |
 | <img alt="logo of project Esprima" src="https://github.com/openjs-foundation/artwork/raw/HEAD/projects/esprima/esprima-icon-color.svg" width="30" /> | [Esprima](https://esprima.org/)            |    | [Contributing Guide](https://github.com/jquery/esprima/blob/HEAD/CONTRIBUTING.md) |  |
```
Comment (ljharb) [inline README.md:40]:
```suggestion
| <img alt="logo of project Esprima" src="https://github.com/openjs-foundation/artwork/raw/HEAD/projects/esprima/esprima-icon-color.svg" width="30" /> | [Esprima](https://esprima.org/)            |    | [Contributing Guide](https://github.com/jquery/esprima/blob/HEAD/CONTRIBUTING.md) |
```

File: README.md [hunk 2]
```diff
@@ -21,64 +21,64 @@ The following projects are official OpenJS Foundation projects. If you are inter
...
+| <img alt="logo of project Express" src="https://github.com/openjs-foundation/artwork/raw/HEAD/projects/express/express-icon-black.svg" width="30" /> | [Express](https://expressjs.com/)          |    | [Contributing Guide](https://expressjs.com/en/resources/contributing.html) |
+| <img alt="logo of project Fastify" src="https://github.com/openjs-foundation/artwork/raw/HEAD/projects/fastify/fastify-icon-black.svg" width="30" /> | [Fastify](https://fastify.io)          |    | [Contributing Guide](https://github.com/fastify/fastify/blob/HEAD/CONTRIBUTING.md) |
 | <img alt="logo of project Globalize" src="https://github.com/openjs-foundation/artwork/raw/HEAD/projects/globalize/globalize-icon-color-dark_background.svg" width="30" /> | [Globalize](https://github.com/jquery/globalize)  |    | [Contributing Guide](https://github.com/globalizejs/globalize/blob/HEAD/CONTRIBUTING.md) | |
```
Comment (ljharb) [inline README.md:43]:
```suggestion
| <img alt="logo of project Globalize" src="https://github.com/openjs-foundation/artwork/raw/HEAD/projects/globalize/globalize-icon-color-dark_background.svg" width="30" /> | [Globalize](https://github.com/jquery/globalize)  |    | [Contributing Guide](https://github.com/globalizejs/globalize/blob/HEAD/CONTRIBUTING.md) |
```

File: README.md [hunk 3]
```diff
@@ -21,64 +21,64 @@ The following projects are official OpenJS Foundation projects. If you are inter
...
+| <img alt="logo of project Grunt" src="https://github.com/openjs-foundation/artwork/raw/HEAD/projects/grunt/grunt-icon-color.svg" width="30" /> | [Grunt](https://gruntjs.com/)              |    | [Contributing Guide](https://github.com/gruntjs/grunt/blob/HEAD/CONTRIBUTING.md) |
+| <img alt="logo of project HospitalRun" src="https://github.com/openjs-foundation/artwork/raw/HEAD/projects/hospitalrun/hospitalrun-icon-color-dark_background.svg" width="30" /> | [HospitalRun](https://hospitalrun.io/)     |    | [Contributing Guide](https://github.com/HospitalRun/hospitalrun/blob/HEAD/.github/CONTRIBUTING.md) |
 | <img alt="logo of project Interledger.js" src="https://github.com/openjs-foundation/artwork/raw/HEAD/projects/interledgerjs/interledgerjs-icon-color.svg" width="30" /> | [Interledger.js](https://interledger.org/) |    | [Contributing Guide](https://github.com/interledger/rfcs/blob/HEAD/CONTRIBUTING.md) |  |
```
Comment (ljharb) [inline README.md:46]:
```suggestion
| <img alt="logo of project Interledger.js" src="https://github.com/openjs-foundation/artwork/raw/HEAD/projects/interledgerjs/interledgerjs-icon-color.svg" width="30" /> | [Interledger.js](https://interledger.org/) |    | [Contributing Guide](https://github.com/interledger/rfcs/blob/HEAD/CONTRIBUTING.md) |
```

File: README.md [hunk 4]
```diff
@@ -21,64 +21,64 @@ The following projects are official OpenJS Foundation projects. If you are inter
...
 | <img alt="logo of project Intern" src="https://github.com/openjs-foundation/artwork/raw/HEAD/projects/intern/intern-icon-color.svg" width="30" /> | [Intern](https://theintern.github.io/)  |    | [Contributing Guide](https://github.com/theintern/intern/blob/HEAD/CONTRIBUTING.md) |  |
```
Comment (ljharb) [inline README.md:47]:
```suggestion
| <img alt="logo of project Intern" src="https://github.com/openjs-foundation/artwork/raw/HEAD/projects/intern/intern-icon-color.svg" width="30" /> | [Intern](https://theintern.github.io/)  |    | [Contributing Guide](https://github.com/theintern/intern/blob/HEAD/CONTRIBUTING.md) |
```

File: README.md [hunk 5]
```diff
@@ -21,64 +21,64 @@ The following projects are official OpenJS Foundation projects. If you are inter
...
 | <img alt="logo of project JerryScript" src="https://github.com/openjs-foundation/artwork/raw/HEAD/projects/jerryscript/jerryscript-icon-color.svg" width="30" /> | [JerryScript](https://jerryscript.net/)    |    | [Contributing Guide](https://github.com/jerryscript-project/jerryscript/blob/HEAD/CONTRIBUTING.md) | |
```
Comment (ljharb) [inline README.md:48]:
```suggestion
| <img alt="logo of project JerryScript" src="https://github.com/openjs-foundation/artwork/raw/HEAD/projects/jerryscript/jerryscript-icon-color.svg" width="30" /> | [JerryScript](https://jerryscript.net/)    |    | [Contributing Guide](https://github.com/jerryscript-project/jerryscript/blob/HEAD/CONTRIBUTING.md) |
```

File: README.md [hunk 6]
```diff
@@ -21,64 +21,64 @@ The following projects are official OpenJS Foundation projects. If you are inter
...
 | <img alt="logo of project Lodash" src="https://github.com/openjs-foundation/artwork/raw/HEAD/projects/lodash/lodash-icon-color.svg" width="30" /> | [Lodash](https://lodash.com/)              |    | [Contributing Guide](https://github.com/lodash/lodash/blob/HEAD/.github/CONTRIBUTING.md) |  |
```
Comment (ljharb) [inline README.md:49]:
```suggestion
| <img alt="logo of project Lodash" src="https://github.com/openjs-foundation/artwork/raw/HEAD/projects/lodash/lodash-icon-color.svg" width="30" /> | [Lodash](https://lodash.com/)              |    | [Contributing Guide](https://github.com/lodash/lodash/blob/HEAD/.github/CONTRIBUTING.md) |
```

File: README.md [hunk 7]
```diff
@@ -21,64 +21,64 @@ The following projects are official OpenJS Foundation projects. If you are inter
...
+| <img alt="logo of project messageformat" src="https://github.com/openjs-foundation/artwork/raw/HEAD/projects/messageformat/messageformat-icon-color.svg" width="30" /> | [messageformat](https://messageformat.github.io/messageformat/)  |    | [Contributing Guide](https://github.com/messageformat/messageformat/blob/HEAD/CONTRIBUTING.md) |
+| <img alt="logo of project Mocha" src="https://github.com/openjs-foundation/artwork/raw/HEAD/projects/mocha/mocha-icon-color.svg" width="30" /> | [Mocha](https://mochajs.org/)           |    | [Contributing Guide](https://github.com/mochajs/mocha/blob/HEAD/.github/CONTRIBUTING.md) |
 | <img alt="logo of project Moment" src="https://github.com/openjs-foundation/artwork/raw/HEAD/projects/moment/moment-icon-color.svg" width="30" /> | [Moment](https://momentjs.com/)            |    | [Contributing Guide](https://github.com/moment/moment/blob/develop/CONTRIBUTING.md) |  |
```
Comment (ljharb) [inline README.md:54]:
```suggestion
| <img alt="logo of project Moment" src="https://github.com/openjs-foundation/artwork/raw/HEAD/projects/moment/moment-icon-color.svg" width="30" /> | [Moment](https://momentjs.com/)            |    | [Contributing Guide](https://github.com/moment/moment/blob/develop/CONTRIBUTING.md) |
```

**Review by ljharb — APPROVED** [review]

**Comment by tobie** [comment]
Hey, @joesepi, is this one ready? If so, can we merge it?

**Review by mhdawson — APPROVED** [review]
LGTM

**Pull request merged**